### PR TITLE
added extra fields as per #89

### PR DESCRIPTION
**Pull request by sareg0**
All of Anika's extra fields have been added, but I would super duper appreciate a second set of eyes on it :)
Many thanks!

### CODE REVIEW

File: app/views/users/edit.html.slim [hunk 1]
```diff
@@ -43,14 +43,10 @@
   = f.input :company_name, label: 'Company name'
   = f.input :company_info, label: 'Company info', hint: 'Where is the office, how much space can you provide, etc.'
 
-  = f.input :bio, label: 'About you', hint: 'You can use basic <a href="http://en.wikipedia.org/wiki/Markdown" target="_blank">Markdown</a> here.'.html_safe
```
Comment (carpodaster) [inline app/views/users/edit.html.slim]: We still need that one. It's for the public user profile. Here's an [example](https://teams.railsgirlssummerofcode.org/users/3) (it's the 'About me' part).

**Comment by sareg0** [comment]
boom! Put it right back :)

File: app/models/user.rb [hunk 2]
```diff
@@ -25,6 +25,15 @@ class User < ActiveRecord::Base
     'coachingcompany' => 'Providing a coaching team from our company',
   }
 
+  MONTHS_LEARNING = [
+    "1-3 ",
```
Comment (carpodaster) [inline app/models/user.rb]: I wouldn't include the trailing whitespace. They'll end up in the database, too

**Comment by carpodaster** [comment]
Hey @sareg0 – thanks for the PR! :candy: 

Can you add the the new fields to `app/views/application_drafts/_student_fields.html.slim` as well (plus updating the `student_params` in `ApplicationDraftsController`) ? Ideally the fields should be shared really, but currently they're separate.

**Comment by carpodaster** [comment]
@sareg shall I add the fields to the `_student_fields` partial?

**Comment by sareg0** [comment]
hey @carpodaster is it ok if I do it in the morning? Or have you taken care of it?

**Comment by carpodaster** [comment]
@sareg0 no, I haven't done anything. Sure thing, do it in the morning – ping me if you have kwistions!

**Pull request merged**